kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/OrderRepository.java

```diff
@@ -216,16 +210,19 @@ private void decrementPendingOrdersCount() {
 		}
 	}
 
-	private void setCompletedOrder(@NonNull Order order) {
-		Log.i(TAG, "setCompletedOrder: " + order);
-		completedOrder.postValue(order);
+	private void updateOrder(@NonNull Order order) {
```

Comment:
to much logic in one method that called updateOrder

Reply:
your suggestion?

Reply:
only called form one place - remove method and move the logic to getOrder->onResponse()